server/sockets/helpers/index.js

```diff
@@ -43,37 +46,35 @@ const updateListOnDashboardAndCohortView = (
     .then(doc => {
       if (doc) {
         const { viewersIds, cohortId } = doc;
-        const dashboardClientExists = dashboardClients.size > 0;
 
-        if (dashboardClientExists) {
-          viewersIds.forEach(id => {
-            const viewerId = id.toString();
-            const list = responseWithList(doc, id);
+        viewersIds.forEach(id => {
+          const viewerId = id.toString();
+          const list = responseWithList(doc, id);
 
-            if (dashboardClients.has(viewerId)) {
-              // send to users that are on the dashboard view
-              socket.broadcast
-                .to(dashboardClients.get(viewerId))
-                .emit(ListActionTypes.CREATE_SUCCESS, list);
-            }
-          });
-        }
+          if (dashboardClients.has(viewerId)) {
+            const { socketId } = dashboardClients.get(viewerId);
+            // send to users that are on the dashboard view
```

Comment:
this comment is unnecessary, we use dashboardClients here so we know that this event will be sendt to users that are on the dashboard